gradle.properties

```diff
@@ -20,9 +20,9 @@ POM_DEVELOPER_URL = https://github.com/dropbox/
 POM_DEVELOPER_EMAIL = api-support@dropbox.com
 
 # Disable Configuration Cache until we test it in this project
-org.gradle.unsafe.configuration-cache=false
+org.gradle.unsafe.configuration-cache=true
 
 # Use AndroidX
 android.useAndroidX=true
 
-org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -Dfile.encoding=UTF-8
```

Comment:
https://docs.oracle.com/javase/9/migrate/
The permanent generation was removed in JDK 8, this option causes gradle to fail in newer JDK versions.